scripts/build_funcs.sh

```diff
@@ -60,7 +60,7 @@ integration() {
         --env SONOBUOY_CLI="$SONOBUOY_CLI" \
         --network host \
         "$BUILD_IMAGE" \
-    go test ${VERBOSE:+-v} -timeout 3m -tags=integration "$GOTARGET"/test/integration/... -run TestConfigmaps
```

Comment:
Accidentally added in a previous commit.

Reply:
This is actually hiding another bug; on fixing that now.

Reply:
💯 No bug, just a timeout issue since we keep adding new tests. A good thing.  See #1311